--- setup.py.old	2023-02-11 06:30:37.125430837 -0600
+++ setup.py	2023-02-11 06:30:53.029588042 -0600
@@ -68,7 +68,6 @@
         ],
     },
     entry_points={
-        "console_scripts": ["dagster-airflow = dagster_airflow.cli:main"],
         # airflow 1.0/2.0 plugin format
         "airflow.plugins": ["dagster_airflow = dagster_airflow.__init__:DagsterAirflowPlugin"],
         # airflow 2.0 provider format
